packages/extension-base/src/services/balance-service/transfer/xcm/polygonBridge.ts

```diff
@@ -0,0 +1,142 @@
+  const polygonBridgeContract = getWeb3Contract(polygonBridgeContractAddress, evmApi, _POLYGON_BRIDGE_ABI);
+  const tokenContract = _getContractAddressOfToken(tokenInfo) || '0x0000000000000000000000000000000000000000'; // FOR Ethereum: use null address
+
+  if (tokenContract !== '0x0000000000000000000000000000000000000000') {
+    throw new Error('Only native token transfer is supported');
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-member-access,@typescript-eslint/no-unsafe-assignment
+  const transferCall: ContractSendMethod = polygonBridgeContract.methods.bridgeAsset(
+    destinationNetwork,
+    recipientAddress,
+    value,
+    tokenContract,
+    true,
+    '0x'
+  );
+  const transferEncodedCall = transferCall.encodeABI();
+
+  const gasDomain = isTestnet ? POLYGON_GAS_INDEXER.TESTNET : POLYGON_GAS_INDEXER.MAINNET;
+  const gasResponse = isTestnet
+    ? await fetch(`${gasDomain}`).then((res) => res.json()) as gasStation
+    : undefined;
+
+  const gasPriceInWei = gasResponse ? (gasResponse.standard * 1e9 + 200000) : undefined;
+  const priority = !gasResponse ? await calculateGasFeeParams(evmApi, evmApi.chainSlug) : undefined;
+
+  const transactionConfig: TransactionConfig = {
+    from: sender,
+    to: polygonBridgeContractAddress,
+    value: value,
+    data: transferEncodedCall,
+    gasPrice: gasPriceInWei || priority?.gasPrice,
+    maxFeePerGas: priority?.maxFeePerGas?.toString(),
+    maxPriorityFeePerGas: priority?.maxPriorityFeePerGas?.toString()
+  };
+
+  const gasLimit = await evmApi.api.eth.estimateGas(transactionConfig).catch(() => 200000);
+
+  transactionConfig.gas = gasLimit.toString();
+
+  return transactionConfig;
+}
+
+export async function _createPolygonBridgeL1toL2Extrinsic (tokenInfo: _ChainAsset, originChainInfo: _ChainInfo, sender: string, recipientAddress: string, value: string, evmApi: _EvmApi): Promise<TransactionConfig> {
+  return createPolygonBridgeTransaction(tokenInfo, originChainInfo, sender, recipientAddress, value, 1, evmApi, false);
+}
+
+export async function _createPolygonBridgeL2toL1Extrinsic (tokenInfo: _ChainAsset, originChainInfo: _ChainInfo, sender: string, recipientAddress: string, value: string, evmApi: _EvmApi): Promise<TransactionConfig> {
+  const isTestnet = originChainInfo.slug === 'polygonzkEvm_cardona';
+
+  return createPolygonBridgeTransaction(tokenInfo, originChainInfo, sender, recipientAddress, value, 0, evmApi, isTestnet);
+}
+
+export async function getClaimPolygonBridge (chainSlug: string, notification: _NotificationInfo, evmApi: _EvmApi) {
+  const polygonBridgeContractAddress = getPolygonBridgeContract(chainSlug);
+  const polygonBridgeContract = getWeb3Contract(polygonBridgeContractAddress, evmApi, _POLYGON_BRIDGE_ABI);
+  const metadata = notification.metadata as ClaimPolygonBridgeNotificationMetadata;
+
+  const isTestnet = chainSlug === 'sepolia_ethereum';
```

Comment:
use COMMON_CHAIN_SLUGS.ETHEREUM_SEPOLIA instead of. Also check others fixed string.